static/app/views/performance/landing/widgets/transforms/transformMetricsToArea.tsx

```diff
@@ -68,6 +68,7 @@ export function transformMetricsToArea<T extends WidgetDataConstraint>(
   const data = groups.map(group => ({
     seriesName: metricsField,
     totals: group.totals[metricsField],
+    aggregation: defined(totalPerBucket) ? 'failure_rate()' : undefined,
```

Comment:
Is this `aggregation` field necessary? I think you can just do

```
seriesName: defined(totalPerBucket) ? 'failure_rate()' : metricField
```

to get the same results without introducing this hack. We'd want to keep the `Series` type closely aligned with ECharts.

Reply:
I did not want to do that because the description in the tooltip would be different

Reply:
From the screenshots, I see that the description in the tooltip currently says `count(transaction.duration)`. Is there a reason to prefer this over changing it to say `failure_rate()`?

As a user, `count(transaction.duration)` doesn't mean anything to me since `count()` is an aggregation that takes no arguments, and it's not clear what `count(transaction.duration)` is counting. Additionally, this PR changes the tooltip value to render as a percentage so leaving the series name as `count(transaction.duration)` doesn't make sense.